website/blog/2020-02-07-announcing-riff-0-5-0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Announcing riff v0.5.0"
+---
+
```

Comment:
things worth calling out that are not discussed:

- kapp is encouraged for installs
- helm is no longer supported for installs
- each runtime (core, knative, streaming) can be installed independently at any time, and uninstalled independently
- Deployers and Processors now consistently use PodTemplateSpec to define containers and volumes
- CRD status now favor object references instead of resource names